README.md

```diff
@@ -1,5 +1,13 @@
 # MetaMask/action-create-release-pr
 
+This action creates a new release branch and PR. It also makes a commit that includes any release preparation that could be easily autoamted, which at the moment includes the version bump and a partial changelog update (using `@metamask/auto-changelog`).
+
+This action is compatible with monorepos, but only if they use workspaces. It uses the `workspaces` property of `package.json` to determine if the repo is a monorepo, and to find each workspace.
```

Comment:
Nit: leading space.

Reply:
I don't see any leading space :eyes: I think this is just how it's shown in the diff